### PR TITLE
Include argument in select_method example

### PR DESCRIPTION
**Pull request by lars-petter-hauge**
**Issue**
 No issue created - user reported

**Approach**
Include necessary argument in the documentation for the `select_method`

### CODE REVIEW

File: python/ecl/grid/ecl_region.py [hunk 1]
```diff
@@ -62,8 +62,12 @@ def select_method(select):
     in i:[0:5] interval:
 
        region = EclRegion( grid , False )
-       region.select_islice(0 , 5)     # Selects all cells with i:[0:5]
-       region.select_jslice(0 , 5)     # Selects all cells with j:[0:5] AND i:[0:5]
+
+       # Select all cells with i:[0:5]:
```
Comment (markusdregi) [inline python/ecl/grid/ecl_region.py]: What is up with the colons?

Comment (lars-petter-hauge) [reply python/ecl/grid/ecl_region.py]: Probably some sort of preference? How about now? Removed some colons and some whitespace

File: python/ecl/grid/ecl_region.py [hunk 2]
```diff
@@ -52,18 +52,22 @@ def select_method(select):
...
 
        region = EclRegion( grid , False )
-       region.select_islice(0 , 5)     # Selects all cells with i:[0:5]
-       region.select_jslice(0 , 5)     # Selects all cells with j:[0:5] AND i:[0:5]
+
+       # Select all cells with i[0:5]:
+       region.select_islice(0, 5)
+
+       # Select all cells with j[0:5] AND i[0:5]:
+       region.select_jslice(0, 5, intersect = True)
```
Comment (markusdregi) [inline python/ecl/grid/ecl_region.py]: If you are fixing white spaces, then maybe `intersect=True` while you're at it?

File: python/ecl/grid/ecl_region.py (hunk 1 above)
Comment (markusdregi) [reply python/ecl/grid/ecl_region.py]: Still don't get what the colons are about, but I guess it is not mission critical anyway.

File: python/ecl/grid/ecl_region.py (hunk 2 above)
Comment (lars-petter-hauge) [reply python/ecl/grid/ecl_region.py]: fixed

**Pull request merged**